src/main/java/com/aws/greengrass/smbridge/clients/IPCUtils.java

```diff
@@ -0,0 +1,104 @@
+/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0 */
+
+package com.aws.greengrass.smbridge.clients;
+
+import software.amazon.awssdk.crt.io.ClientBootstrap;
+import software.amazon.awssdk.crt.io.EventLoopGroup;
+import software.amazon.awssdk.crt.io.SocketOptions;
+import software.amazon.awssdk.eventstreamrpc.EventStreamRPCConnection;
+import software.amazon.awssdk.eventstreamrpc.EventStreamRPCConnectionConfig;
+import software.amazon.awssdk.eventstreamrpc.GreengrassConnectMessageSupplier;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+
+
+public final class IPCUtils {
+    // Port number is not used in domain sockets.
+    // It is ignored but the field needs to be set when creating socket connection
+    public static final int DEFAULT_PORT_NUMBER = 8033;
+    private static EventStreamRPCConnection clientConnection = null;
+
+    private IPCUtils() {
+
+    }
+
+    /**
+     * In this package, called by the SMClient class to use with IPC to find custom port for Stream Manager
```

Comment:
Is this component not run as a plugin? If it is running as a plugin, it need not go over the IPC.

Reply:
Can I sync with you Monday morning about how to discover the custom port otherwise? This is currently what is throwing an error in the startup() phase of the component...

Reply:
Adding to other PR, so resolving this convo and deleting the IPCUtils file